### PR TITLE
Add use flag to set boot message verbosity level

### PR DESCRIPTION
**Pull request by ksorensen**
Signed-off-by: Klaus Henning Sorensen <klaus.sorensen@prevas.dk>

### CODE REVIEW

File: recipes/busybox/busybox-configure.inc [hunk 1]
```diff
@@ -374,6 +374,15 @@ RECIPE_FLAGS += "busybox_ntpd_splashutils_progress busybox_ntpd_splashutils_msg"
 SPLASHUTILS_INITSCRIPTS:>USE_busybox_ntpd += " busybox_ntpd"
 DEFAULT_USE_busybox_ntpd_splashutils_msg = "\"Starting Busybox ntpd\""
 
+# USE flag:
+RECIPE_FLAGS += "busybox_rcdefault_verbose"
+BUSYBOX_RCDEFAULT_VERBOSE_CONFIGURATION = "very"
+BUSYBOX_RCDEFAULT_VERBOSE_CONFIGURATION:USE_busybox_rcdefault_verbose = "${USE_busybox_rcdefault_verbose}"
+DO_CONFIGURE_PREFUNCS:>USE_busybox_rcdefault_verbose += " do_configure_busybox_rcdefault_verbose"
```
Comment (esben) [inline recipes/busybox/busybox-configure.inc]: You don't need to use '+=' here, '=' would be fine.

**Comment by esben** [comment]
LGTM :+1:

**Comment by ksorensen** [comment]
Changed unnecessary += to = and force pushed.

**Comment by kimrhh** [comment]
merged, thanks

**Pull request merged**